react/features/base/tracks/actions.js

```diff
@@ -172,6 +173,20 @@ export function destroyLocalTracks() {
     };
 }
 
+/**
+ * Signals that the local participant is ending screensharing or beginning the
+ * screensharing flow.
+ *
+ * @returns {{
+ *     type: TOGGLE_SCREENSHARE,
+ * }}
+ */
+export function toggleScreenshare() {
+    return {
+        type: TOGGLE_SCREENSHARE
+    };
+}
```

Comment:
The current toolbar does pubsub out to APP.conference. Instead, hide that detail from the new toolbar by using middleware to communicate out to APP.conference.